doc/man3/SSL_CTX_set_options.pod

```diff
@@ -170,6 +170,12 @@ RFC7366 Encrypt-then-MAC option on TLS and DTLS connection.
 If this option is set, Encrypt-then-MAC is disabled. Clients will not
 propose, and servers will not accept the extension.
 
+=item SSL_OP_NO_RENEGOTIATION
+
+Disable all renegotiation. Do not accept renegotiation requests.
+
+This also disables KeyUpdate messages in TLSv1.3.
```

Comment:
Hrm. You all can certainly define APIs how you like them, but I'm curious why this semantic. I can see lots of reasons to forbid renego (and indeed we do so), but KeyUpdate is a lot more tame (we don't intend to forbid it). It doesn't trigger a new handshake, change high-level connection properties, or jam up read and write against each other.